hail/python/hail/expr/types.py

```diff
@@ -835,6 +836,10 @@ def _eq(self, other):
                 and all(self[f] == other[f] for f in self._fields))
 
     def _pretty(self, l, indent, increment):
+        if not self.items():
```

Comment:
prefer `self._fields` for clarity

Reply:
I want the if to match the loop below.  I changed them both to `self._field_types`.